ibis/expr/schema.py

```diff
@@ -369,9 +369,12 @@ def coerce_to_dataframe(
     if isinstance(data, pd.DataFrame):
         result = data
     elif isinstance(data, pd.Series):
-        num_cols = len(data.iloc[0])
-        series = [data.apply(lambda t: t[i]) for i in range(num_cols)]
-        result = pd.concat(series, axis=1)
+        if not len(data):
+            result = pd.concat([data], axis=1)
```

Comment:
Why not `result = data.to_frame()`?

Reply:
Good point, updated to use `to_frame` instead.